apps_product_creator/models/product_product.py

```diff
@@ -57,6 +79,17 @@ class ProductProduct(models.Model):
         store=True,
     )
 
+    @api.depends('odoo_module_version_id',
+                 'odoo_module_version_id.description_rst_html')
+    @api.multi
+    def _compute_rst_html_desc(self):
+        for product in self:
+            rst_desc = product.odoo_module_version_id.description_rst_html
```

Comment:
I assume that `description_rst_html` will already contain either html if exist or rst as fallback.
Do you confirm?

Reply:
@elicoidal  yes it will. confirmed with https://github.com/OCA/server-tools/blob/11.0/base_fontawesome/ module and it is working